code/modules/mob/living/silicon/laws.dm

```diff
@@ -90,9 +91,13 @@
 	post_lawchange(announce)
 
 /mob/living/silicon/proc/make_laws()
-	laws = new /datum/ai_laws
-	laws.set_laws_config()
-	laws.associate(src)
+	if(SSticker.current_state < GAME_STATE_PLAYING)
+		addtimer(CALLBACK(src, /mob/living/silicon/.proc/make_laws), 3 SECONDS) // game data should be established before game started, then the data can be used to law setup
+	else
+		laws = new /datum/ai_laws
+		laws.set_laws_config()
+		laws.associate(src)
```

Comment:
because laws are created before roundstart, while the racimov laws need game data after roundstart, I had to make `make_laws()` delayed so that it can find data.
There was nothing particularly going bad at a test.

<details>
<summary>See the late law establish result</summary>

![image](https://user-images.githubusercontent.com/87972842/175858882-4ed6ae35-3514-4233-9a71-2c430042b67a.png)
</detail>